home/migrations/0083_alter_contentpage_whatsapp_body.py

```diff
@@ -31,7 +31,7 @@ def list_string_to_action(block_value):
 
 class ListStringToActionOperation(BaseBlockOperation):
     def apply(self, block_value):
-        list_string_to_action(block_value)
+        list_string_to_action(block_value)  # this is wrong, there should be a return
```

Comment:
Does this not imply data loss then? Should we maybe fix it before it gets deployed to production?